$NetBSD: patch-aa,v 1.11 2018/01/28 19:58:17 wiz Exp $

Shell portability fix: see
https://savannah.nongnu.org/bugs/index.php?22222
Ensure asm instructions are checked correctly.

--- configure.orig	2017-11-17 23:09:24.000000000 +0000
+++ configure
@@ -15484,7 +15484,7 @@ else
 fi
 
 
-if test x$PKG_CONFIG == xno && test "x${multithread}" != xno ; then
+if test x$PKG_CONFIG = xno && test "x${multithread}" != xno ; then
 
 
 
@@ -16579,10 +16579,10 @@ main ()
                  int n   = 0;
                  int eax = 0;
                  int edx = 0;
-                 __asm__(
+                 __asm__ volatile (
                      "xgetbv     \n"
                      "vzeroupper  "
-                     : "=a"(eax), "=d"(edx) : "c"(n) : );
+                     : "=a"(eax), "=d"(edx) : "c"(n) );
              #else
                  #error No GCC style inline asm supported for AVX instructions
              #endif
@@ -16605,7 +16605,7 @@ fi
 rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $gcc_inline_asm_avx" >&5
 $as_echo "$gcc_inline_asm_avx" >&6; }
-if test "x${gcc_inline_asm_avx}" == xyes ; then
+if test "x${gcc_inline_asm_avx}" = xyes ; then
     $as_echo "#define OPENEXR_IMF_HAVE_GCC_INLINE_ASM_AVX 1" >>confdefs.h
 
 fi
@@ -16636,7 +16636,7 @@ fi
 rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $sysconf_nproc" >&5
 $as_echo "$sysconf_nproc" >&6; }
-if test "x${sysconf_nproc}" == xyes ; then
+if test "x${sysconf_nproc}" = xyes ; then
     $as_echo "#define OPENEXR_IMF_HAVE_SYSCONF_NPROCESSORS_ONLN 1" >>confdefs.h
 
 fi
@@ -16750,7 +16750,7 @@ LIB_SUFFIX=""
 lib_suffix_valid="no"
 
 lib_namespace="Imf"
-if test "x${library_namespace_versioning}" == xyes ; then
+if test "x${library_namespace_versioning}" = xyes ; then
     cat >>confdefs.h <<_ACEOF
 #define OPENEXR_IMF_INTERNAL_NAMESPACE Imf_${OPENEXR_VERSION_API}
 _ACEOF
@@ -16761,7 +16761,7 @@ _ACEOF
     lib_namespace="Imf_${OPENEXR_VERSION_API}"
     LIB_SUFFIX="${OPENEXR_VERSION_API}"
     lib_suffix_valid="yes"
-elif test "x${library_namespace_versioning}" == xno ; then
+elif test "x${library_namespace_versioning}" = xno ; then
     cat >>confdefs.h <<_ACEOF
 #define OPENEXR_IMF_INTERNAL_NAMESPACE Imf
 _ACEOF
@@ -16782,7 +16782,7 @@ _ACEOF
 fi
 
 
-if test "x${lib_suffix_valid}" == xyes ; then
+if test "x${lib_suffix_valid}" = xyes ; then
 LIB_SUFFIX_DASH="-${LIB_SUFFIX}"
 
  if true; then
@@ -16817,7 +16817,7 @@ else
 fi
 
 
-if test "x${custom_usr_namespace}" == xyes ; then
+if test "x${custom_usr_namespace}" = xyes ; then
     { $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: Enabling 'custom user namespace' requires an additional argument, reverting to 'Imf'" >&5
 $as_echo "$as_me: WARNING: Enabling 'custom user namespace' requires an additional argument, reverting to 'Imf'" >&2;}
     cat >>confdefs.h <<_ACEOF
@@ -16825,7 +16825,7 @@ $as_echo "$as_me: WARNING: Enabling 'cus
 _ACEOF
 
     usr_namespace="Imf"
-elif test "x${custom_usr_namespace}" == xno ; then
+elif test "x${custom_usr_namespace}" = xno ; then
     cat >>confdefs.h <<_ACEOF
 #define OPENEXR_IMF_NAMESPACE Imf
 _ACEOF
@@ -19539,7 +19539,7 @@ enable large stack optimizations        
 internal library namespace                      $lib_namespace
 user-client namespace                           $usr_namespace" >&6; }
 
-if test "x$build_osxuniversal" == xyes; then
+if test "x$build_osxuniversal" = xyes; then
 { $as_echo "$as_me:${as_lineno-$LINENO}: result:
 build OS X universal binaries                   $build_osxuniversal" >&5
 $as_echo "
